test/DynamoCoreWpfTests/ViewExtensions/DocumentationBrowserViewExtensionTests.cs

```diff
@@ -865,10 +870,28 @@ private List<MenuItem> GetDocsMenuItems()
                 .ToList();
         }
 
+        private void WaitForWebView2Initialization(DocumentationBrowserView docView = null)
+        {
+            if (docView == null)
+            {
+                var docsTab = GetDocsTabItem();
+                docView = docsTab.Content as DocumentationBrowserView;
+            }
+            
+            // Wait for the DocumentationBrowserView webview2 control to finish initialization
+            DispatcherUtil.DoEventsLoop(() =>
```

Comment:
Hopefully not all the time. Only if the webview2 control is expected to be visible during the test

Reply:
I will try to add some sort of docs somewhere

Reply:
It would be a really nice addition with a custom analyzer using roslyn. We could detect usage of an API that would show the WebView control, and add a compiler error if we don't see this method used... anyway, for another day.